src/systems/inter_mesh_projection.C

```diff
@@ -110,24 +105,19 @@ void InterMeshProjection::project_system_vectors()
       from_system.get_vector(vec_name).localize(*current_vector_proxy);
 
       // Construct a MeshFunction for the current component
-      MeshFunction * mesh_func =
-        new MeshFunction(from_system.get_equation_systems(), *current_vector_proxy,
-                         from_system.get_dof_map(), variables_vector);
-
+      std::unique_ptr<MeshFunction> mesh_func(new MeshFunction(from_system.get_equation_systems(), *current_vector_proxy,
```

Comment:
Here we're having to loop over all the vectors being projected, but since we don't need each MeshFunction outside the loop, we should be able to just keep things on the stack again, right?